src/__tests__/JWT/CommonSignerTest/CommonSignerTest.ts

```diff
@@ -0,0 +1,205 @@
+import * as jwt from 'jsonwebtoken'
+import * as u8a from 'uint8arrays'
+import { Extensible, ServiceEndpoint } from 'did-resolver'
+
+// redeclare non-exported interface from did-resolver
+interface JsonWebKey extends Extensible {
+  alg?: string
+  crv?: string
+  e?: string
+  ext?: boolean
+  key_ops?: string[]
+  kid?: string
+  kty: string
+  n?: string
+  use?: string
+  x?: string
+  y?: string
+}
+
+interface privateJsonWebKey extends JsonWebKey {
+  d?: string
+}
+
+interface VerificationMethodLegacyCommonSigner {
+  id?: string
+  type: string
+  owner?: string
+  publicKey?: string
+  publicKeyBase58?: string
+  publicKeyBase64?: string
+  publicKeyJwk?: JsonWebKey
+  publicKeyHex?: string
+  publicKeyMultibase?: string
+  blockchainAccountId?: string
+  ethereumAddress?: string
+}
+
+interface VerificationMethodCommonSigner {
```

Comment:
Why is this definition necessary since there already exists a `VerificationMethod` definition?

Reply:
> There were some other things in there like (https://github.com/bshambaugh/did-jwt/blob/master/src/__tests__/JWT/CommonSignerTest/CommonSignerTest.ts#L24-L75) that were added to avoid the use of **any** in the typescript file. This might need to be refactored. I guess I didn't realize how scary this was going to get.

export interface VerificationMethod {
  id: string
  type: string
  controller: string
  publicKeyBase58?: string
  publicKeyBase64?: string
  publicKeyJwk?: JsonWebKey
  publicKeyHex?: string
  publicKeyMultibase?: string
  blockchainAccountId?: string
  ethereumAddress?: string
}

https://github.com/decentralized-identity/did-resolver/blob/master/src/resolver.ts#L88-L99

I recall that the legacy versions were not matching the https://www.w3.org/TR/did-core/ spec.

Reply:
The reason I'm asking this is because the interface is already exported by did-resolver and you can extend it for the legacy use-case tests, making it more obvious which bits are considered legacy.

This is more a matter of preference, honestly, since we're dealing with tests here.